pkg/alerts/alertsqlite/alerts_sqlite.go

```diff
@@ -376,9 +376,14 @@ func (p Sqlite) GetAllAlerts() ([]alertutils.AlertInfo, error) {
 		log.Errorf("getAllAlerts: unable to begin transaction, err: %+v", err)
 		return nil, err
 	}
-	// #todo: Add silence_minutes back in actual fix
-	// sqlStatement := "SELECT alert_id, alert_name, state, create_timestamp, contact_id, labels, silence_minutes FROM all_alerts;"
-	sqlStatement := "SELECT alert_id, alert_name, state, create_timestamp, contact_id, labels FROM all_alerts;"
+	sqlSta := "ALTER TABLE all_alerts ADD COLUMN silence_minutes INTEGER DEFAULT 0;"
+	_, errorAlter := tx.Exec(sqlSta)
+	if err != nil {
+		log.Errorf("getAllAlerts: Error adding column: %v, err: %+v", sqlSta, errorAlter)
+		return nil, err
+	}
```

Comment:
Will this cause issues if the column already exists? So then we'd want to check if it exists and only add it if it doesn't, right?

Reply:
From our meeting it looks like this doesn't cause any issue when the column already exists